.vitepress/config.ts

```diff
@@ -93,6 +93,7 @@ const sidebars = (): DefaultTheme.SidebarItem[] => [
     text: 'Guides',
     collapsed: true,
     items: [
+      { text: 'Create-hono', link: '/docs/guides/create-hono' },
```

Comment:
This is a matter of taste, but I prefer to use `create-hono` instead of `Create-hono`.